python/paddle/nn/functional/common.py

```diff
@@ -1889,7 +1889,7 @@ def linear(x, weight, bias=None, name=None):
                 type='elementwise_add',
                 inputs={'X': [tmp], 'Y': [bias]},
                 outputs={'Out': [res]},
-                attrs={'axis': len(x.shape) - 1},
+                attrs={'axis': -1},
```

Comment:
why only support axis = -1

Reply:
Because in phi standard we only allow axis=-1